doc/motor_extensions.py

```diff
@@ -129,6 +129,13 @@ def get_motor_attr(motor_class, name, *defargs):
     attribute. While we're at it, store some info about each attribute
     in the global motor_info dict.
     """
+    try:
+        attr = safe_getattr(motor_class, name)
+    except AttributeError:
+        if name == '__annotations__':
+            return {}
+        raise
+
     attr = safe_getattr(motor_class, name)
```

Comment:
Let's change this to `attr = safe_getattr(motor_class, name, *defargs)` instead of catching AttributeError.

Reply:
Done.